### PR TITLE
Allow zipped vcf as input

### PR DESCRIPTION
**Pull request by jwcodee**

### CODE REVIEW

File: ntedit.cpp [hunk 1]
```diff
@@ -2064,6 +2067,26 @@ readAndCorrect(
...
+	// cout << tokens.size() << " numtoken\n";
+	if (tokens.size() >= 8) {
+		std::ostringstream id;
+		id << tokens[0] << ">" << tokens[3] << tokens[1] << tokens[4];
+		std::string varid = id.str();
+		var_map[varid] = tokens[7];
+		// Print results
+		// for (auto t : tokens) {
+		//	std::cout << t << std::endl;
+		// }
```
Comment (lcoombe) [inline ntedit.cpp:2086]: Just delete the commented-out code

**Review by lcoombe — COMMENTED** [review]
Looks good!

I can see this line in the README: 
```
Make sure you decompress vcf.gz before use (e.g., unpigz/gunzip clinvar_20230813.vcf.gz)
```
Probably should delete it with this PR

File: ntedit.cpp [hunk 2]
```diff
@@ -2320,30 +2343,30 @@ main(int argc, char** argv) // NOLINT
...
+			// Convert streambuf to istream
+			std::istream instream(&inbuf);
+			// Iterate lines
+			if (myfile.is_open()) {
+				while (std::getline(instream, line)) {
+					vcf_entry_to_map(line, clinvar);
+				}
+				myfile.close();
+			} else {
+				std::cout << "Unable to open file";
```
Comment (parham-k) [inline ntedit.cpp:2363]: Please write error messages to `std::cerr` to keep consistent and append an `std::endl` to flush the stream.

File: ntedit.cpp (hunk 1 above)
Comment (jwcodee) [reply ntedit.cpp]: done

File: ntedit.cpp (hunk 2 above)
Comment (jwcodee) [reply ntedit.cpp]: done

**Pull request merged**